content/automate/ManageIQ/Transformation/TransformationHosts/StateMachines/TransformationHostRole.class/check.yaml

```diff
@@ -0,0 +1,15 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name: 
+    name: Check
+    inherits: 
+    description: 
+  fields:
+  - State1:
+      value: "/Transformation/TransformationHosts/${#transformation_host_type}/RoleCheck"
+      on_error: "/Transformation/TransformationHosts/${#transformation_host_type}.MarkAsDisabled"
+  - State2:
+      value: "/Transformation/TransformationHosts/${#transformation_host_type}.MarkAsEnabled"
```

Comment:
Same here.

Reply:
@fdupont-redhat Does this need a METHOD:: since its a call to a class method

Reply:
@fdupont-redhat Also is this TagAsEnabled?

Reply:
You're right. It is TagAsDisabled. I have renamed it.

@mkanoor `METHOD::` is not needed because `on_error` expects method(s). I don't know how it is coded internally, but experimentally it works without it. If you think I should make it explicit, I'll change it.